docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,87 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
+
+1. Generate AWS Long Term Access Key
+
+* For steps on creating an AWS Access Key go to [AWS Long Term Access Key](../operator-docs/aws-long-term-access-key).
+* Copy the AWS Long Term Access Key.
+
+2. Add a Collection
+
+* Open another tab in your browser and go to your Earthdata Git repository URL or where your collection data files are located.
```

Comment:
We have a description of collection config params here: https://nasa.github.io/cumulus/docs/data-cookbooks/setup#collections

We should also say they can manually configure their collections if they don't have them located somewhere.

Reply:
@laurenfrederick This content is updated with manually option included.